### ISSUE TITLE
Cannot change ID (non self hosted)

### ISSUE THREAD

**Issue by ELkoos** [rustdesk/rustdesk#3148]
Hello. I am trying to change ID after I had cloned my device. This is default installation on Windows 10 Pro. Tried both config files location (C: Windows and users appdata), to no avail. Uninstalling and reinstalling doesn't help too. Please advise what can I do with it? Even if I change ID to 0 as suggested in other thread in C:Windows toml file, it goes back to same number.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#change-id-addressbookapilogin-and-web-console

**Comment by ELkoos** [comment]
This doesn't resolve my issue.

**Comment by rustdesk** [comment]
Not allowed to change id anymore in night version.

**Comment by ELkoos** [comment]
OK, let me rephrase. How and where is ID stored now? Which files are responsible for recreating ID to same ID, even without connecting to relay. Version 1.1.9 is considered nightly?
